app/src/userpages/components/ProfilePage/index.jsx

```diff
@@ -75,6 +72,7 @@ export class ProfilePage extends Component<Props, State> {
         const { saving } = this.state
         return (
             <CoreLayout
+                unpadded
```

Comment:
Is this option necessary.. style could be overridden with eg. `styled-components`?

Reply:
Ha, interesting, maybe. I'll give it a try!

Reply:
@juhah in order to use styled components I'd have to expose that `Content` wrapper and style it for profile and stream pages separately. Don't quite like it.

What I did instead:
- I don't apply any padding to "content" wrapper by default,
- I apply it to individual `Layout/Core` instances across the app (using `.pad` from `core.pcss`)

This way we explicitly indicate layouts that come with padding thus avoid unwanted padding spil. Does it make sense?